service-runner.js

```diff
@@ -100,6 +102,7 @@ ServiceRunner.prototype._sanitizeConfig = function(conf, options) {
         // use the number of CPUs
         conf.num_workers = os.cpus().length;
     }
+    conf.timeout = conf.timeout || 1000;
```

Comment:
I concur. Something making it clear that it's about workers could also be useful, like `worker_heartbeat_timeout`.